### PR TITLE
Fix formatting of commands in CIS blog post

### PR DESCRIPTION
**Pull request by zubron**
Signed-off-by: Bridget McErlean <bmcerlean@vmware.com>

### CODE REVIEW

File: site/_posts/2019-11-18-cis-benchmark-plugin.md [hunk 1]
```diff
@@ -158,10 +158,10 @@ For the master nodes, we need to do three things:
...
 
 ```
-$ cat kub-bench-worker.yaml | \
-  sed ‘/kube-bench-worker/kube-bench-master/’ | \
-  sed ‘/- kube-bench/- kube-bench master/’ | \
-  sed ‘/Exists/DoesNotExist/’ > kube-bench-master.yaml
+$ cat kube-bench-worker.yaml | \
+  sed 's/kube-bench-worker/kube-bench-master/g' | \
+  sed 's/- kube-bench/- kube-bench master/g' | \
+  sed 's/Exists/DoesNotExist/g' > kube-bench-master.yaml
```
Comment (johnSchnake) [inline site/_posts/2019-11-18-cis-benchmark-plugin.md]: We accidentally swapped these around; should be `s/DoesNotExists/Exists/g`

File: site/_posts/2019-11-18-cis-benchmark-plugin.md [hunk 2]
```diff
@@ -94,9 +94,9 @@ We can chain these desired commands together as a single bash command and then s
 $ sonobuoy gen plugin \
     --name kube-bench-worker \
     --image=schnake/kube-bench:v0.2.0-demo \
-    --cmd=/bin/sh
-    --args=-c
-    --args=”kube-bench --version 1.13 --outputfile /tmp/results/output.xml --junit ; echo -n /tmp/results/output.xml > /tmp/results/done” \
+    --cmd=/bin/sh \
```
Comment (johnSchnake) [inline site/_posts/2019-11-18-cis-benchmark-plugin.md]: Also add a line with `--format junit \`; that failed to get added into the post.

File: site/_posts/2019-11-18-cis-benchmark-plugin.md [hunk 3]
```diff
@@ -94,9 +94,11 @@ We can chain these desired commands together as a single bash command and then s
 $ sonobuoy gen plugin \
     --name kube-bench-worker \
     --image=schnake/kube-bench:v0.2.0-demo \
-    --cmd=/bin/sh
-    --args=-c
-    --args=”kube-bench --version 1.13 --outputfile /tmp/results/output.xml --junit ; echo -n /tmp/results/output.xml > /tmp/results/done” \
+    --type=DaemonSet \
```
Comment (zubron) [inline site/_posts/2019-11-18-cis-benchmark-plugin.md]: The type should also be DaemonSet, right?

Comment (johnSchnake) [reply site/_posts/2019-11-18-cis-benchmark-plugin.md]: Yes; thanks! Sorry these were such a mess; it was a mix of manually making the files and then writing the commands afterwards.

**Pull request merged**